frappe/workflow/doctype/workflow_state/workflow_state.json

```diff
@@ -1,6 +1,7 @@
 {
  "actions": [],
  "allow_import": 1,
+ "allow_rename": 1,
```

Comment:
Workflow rename is not as simple, it will break existing documents.

Reply:
workflow state -> workflow -> affected documents. 

This should technically work as they're link fields :sweat_smile: 


Needs to be tested though. We have had many complaints in past about stale workflow state names breaking workflows.

Reply:
maybe that could be solved by setting a pop-up message. Just like when removing workflow states from Workflow you get something hinting that you will need to clean up afterwards.